rdt/transformers/pii/anonymizer.py

```diff
@@ -132,7 +132,9 @@ def get_supported_sdtypes(cls):
             list:
                 Accepted input sdtypes of the transformer.
         """
-        unsupported_sdtypes = {'numerical', 'datetime', 'categorical', 'boolean', 'text', None}
+        unsupported_sdtypes = {
+            'numerical', 'datetime', 'categorical', 'boolean', 'text', None, 'id'
```

Comment:
Do we need to add `id` here? My understanding from the discussion was that `RDT` doesn't support the `id` sdtype.

Reply:
Yes good catch thank you, I forgot about it, done in [f3638cc](https://github.com/sdv-dev/RDT/pull/680/commits/f3638cc1b93f24fd7450f2ddd8fa85677ad1fc0a)